packages/lifecycle/spec/LifeCycleBehaviors.ts

```diff
@@ -10,7 +10,9 @@ import {
 
 import {
   LifeCycle
-} from '../dist'
+} from '../src'
+
+import { assert } from 'chai';
```

Comment:
import order should have relative imports after vendor/absolute i.e. 'chai' before '../src'

Reply:
Changed in https://github.com/Auge19/tko/pull/4 . It is only a stylistic pattern or is there another reason?